README.md

```diff
@@ -146,32 +148,47 @@ Any headers starting with `Grpc-` will be prefixed with an `X-`, this is because
 
 All other headers will be converted to metadata as is.
 
-### CORS Configuration.
+### CORS Configuration
 
 You can configure [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for your gateway through the
 configuration. This will allow your gateway to receive requests from different origins.
 
 There are four values:
 
-- `cors.allow-origin`: Value to set for Access-Control-Allow-Origin header.
-- `cors.allow-credentials`: Value to set for Access-Control-Allow-Credentials header.
-- `cors.allow-methods`: Value to set for Access-Control-Allow-Methods header.
-- `cors.allow-headers`: Value to set for Access-Control-Allow-Headers header.
+* `cors.allow-origin`: Value to set for Access-Control-Allow-Origin header.
+* `cors.allow-credentials`: Value to set for Access-Control-Allow-Credentials header.
+* `cors.allow-methods`: Value to set for Access-Control-Allow-Methods header.
+* `cors.allow-headers`: Value to set for Access-Control-Allow-Headers header.
 
     For CORS, you will want to configure your `cors.allow-methods` to be the HTTP verbs set in your proto (i.e. `GET`, `PUT`, etc.), as well as `OPTIONS`, so that your service can handle the [preflight request](https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request).
 
     If you are not using CORS, you can leave these configuration values at their default, and your gateway will not accept CORS requests.
+
 ### GRPC Client Configuration
-- `grpc.max-call-recv-msg-size`: Sets the maximum message size in bytes the client can receive.
-- `grpc.max-call-send-msg-size`: Sets the maximum message size in bytes the client can send.
-###  Other Response Headers
+
+* `grpc.max-call-recv-msg-size`: Sets the maximum message size in bytes the client can receive.
+
+* `grpc.max-call-send-msg-size`: Sets the maximum message size in bytes the client can send.
+
+### Other Response Headers
 
 You can configure additional headers to be sent in the HTTP response.  
 Set environment variable with prefix `<SERVICE>_RESPONSE-HEADERS_` (e.g `SOMESERVICE_RESPONSE-HEADERS_SOME-HEADER-KEY`).  
 You can also set headers in the your configuration file (e.g `response-headers.some-header-key`)
 
-### Proto names format
-By default, gRPC gateway will return proto names as they are in the proto messages. You can change this behavior by setting `gateway.use-json-names: true` and the gateway will use camelCase JSON names.
+### Marshalling options
+
+#### Proto names format
+
+By default, `gen-grpc-gateway` will return proto names as they are in the proto messages. You can change this behavior by setting `gateway.use-json-names: true` and the gateway will use camelCase JSON names.
+
+#### Unpopulated fields
+
+By default, `gen-grpc-gateway` will not emit unpopulated fields. You can change this behavior by setting `gateway.emit-unpopulated: true` and the gateway will populate these fields with default values.
```

Comment:
Hi, is that for generating `omitempty` JSON option?
https://golang.org/pkg/encoding/json/#Marshal

Reply:
yes, the default behavior has changed in grpc-gateway v2 (https://github.com/grpc-ecosystem/grpc-gateway/blob/v2.0.0/docs/_docs/v2-migration.md#we-now-emit-default-values-for-all-fields) and this PR just sets old behavior as default.